Amqp/src/Akka.Streams.Amqp/Akka.Streams.Amqp.csproj

```diff
@@ -1,9 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <AssemblyTitle>Akka.Streams.Amqp</AssemblyTitle>
+    <Description>AMQP adapter for Akka.NET Streams</Description>
+    <Copyright>Copyright © 2017 Akka.NET Contrib</Copyright>
+    <Authors>Akka.NET Contrib</Authors>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <PackageTags>akka;actors;streams;AMQP;RabbitMQ</PackageTags>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageIconUrl>http://getakka.net/images/akkalogo.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/AkkaNetContrib/Alpakka</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/AkkaNetContrib/Alpakka/blob/dev/LICENSE</PackageLicenseUrl>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Akka.Streams" Version="1.3.2" />
     <PackageReference Include="RabbitMQ.Client" Version="5.0.1" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
Is `System.ValueTuple` required at this point?

Reply:
Yes. After I added .net 4.6 target the project failed to compile without this reference.